package.json

```diff
@@ -85,14 +84,14 @@
     "generate-svg": "gulp test-render && node test/utils/generate-svg.js",
     "style": "gulp style",
     "gulp": "gulp",
-    "postinstall": "guppy pre-commit || exit 0",
-    "prepare": "npm run build"
+    "postinstall": "npm install --ignore-scripts --production=false && npm run build && guppy pre-commit || exit 0"
```

Comment:
Rely on `postinstall` to build the artfiacts.
This'll first download all the dependencies (including dev), and then eventually run `gulp build`.

Caveat: Running `npm install` which cause `postinstall` to run again, which leads to a never ending loop. Worked around this by using `--ignore-scripts` flag, although this also stops scripts from running for the dependencies.